clippy_lints/src/doc/needless_doctest_main.rs

```diff
@@ -69,9 +69,7 @@ pub fn check(
                                 body: Some(block),
                                 ..
                             }) if ident.name == sym::main => {
-                                if !ignore {
-                                    get_test_spans(&item, *ident, &mut test_attr_spans);
-                                }
+                                get_test_spans(&item, *ident, &mut test_attr_spans);
```

Comment:
Is there a reason for removing the `if` here? Won't this now emit a warning for `test_attr_in_doctest` for something like
```rs
/// ```no_run
/// #[test]
/// fn main() { todo!() }
/// ```
fn test() {}
```
?
(Which seems wrong since `test_attr_in_doctest`'s advice in the lint description is to annotate the doctest with `no_run` if intended, which it is here)

Reply:
Little oversight, removed by accident.